libs/ledger-live-common/src/families/ethereum/walletApiAdapter.ts

```diff
@@ -1,44 +1,71 @@
 import { EthereumTransaction as WalletAPIEthereumTransaction } from "@ledgerhq/wallet-api-core";
+import { Account } from "@ledgerhq/types-live";
 import {
   AreFeesProvided,
   ConvertToLiveTransaction,
   GetWalletAPITransactionSignFlowInfos,
 } from "../../wallet-api/types";
-import { Transaction } from "./types";
+import { Transaction as EthTx } from "./types";
+import { Transaction as EvmTx } from "../evm/types";
+import { isTokenAccount } from "../../account";
 
 const CAN_EDIT_FEES = true;
 
+type Transaction = EthTx | EvmTx;
+
 const areFeesProvided: AreFeesProvided<WalletAPIEthereumTransaction> = (tx) =>
-  !!(tx.gasLimit || tx.gasPrice);
+  !!(tx.gasLimit || tx.gasPrice || tx.maxFeePerGas || tx.maxPriorityFeePerGas);
```

Comment:
Maybe we should be a little more strict with partial provided fee info here ? 

Either (tx.gasLimit && tx.gasPrice) || (tx.gasLimit && tx.maxFeePerGas && tx.maxPriorityFeePerGas) should be true ? That's an open question on if we want to support partial tx fee hints

Reply:
Good point, I applied.